src/components/Connections/CreateConnectionForm.tsx

```diff
@@ -490,6 +491,7 @@ const CreateConnectionForm = ({
                           <Switch
                             data-testid={`stream-sync-${idx}`}
                             checked={stream.selected}
+                            disabled={stream.syncMode === "incremental" ? true : false}
```

Comment:
no need to enforce this, if they want to deselect a stream they should be able to do so without changing the sync mode first

Reply:
- Got it. I misunderstood overwrite as changing the sync mode (toggle button).
- Overwrite is the option in destination.

Reply:
-This is removed. User can deselect a stream, when incremental is selected or not.